app/views/personas/index.html.erb

```diff
@@ -29,6 +29,7 @@
           row.with_cell(header: true, text: "Change work history")
           row.with_cell(header: true, text: "Reverse decisions")
           row.with_cell(header: true, text: "Support console")
+          row.with_cell(header: true, text: "Verify Permissions")
```

Comment:
```suggestion
          row.with_cell(header: true, text: "Verify applicants")
```

Since that's a better description of what the permission gives you access to do, and we're already within a cell group titled "Permissions". Also, I think we'll need to increase the `colspan`s above to include this column.